cypress/e2e/hub/collections.cy.ts

```diff
@@ -57,49 +58,106 @@ describe('Collections- List View', () => {
+    const collection =
+      'hub_e2e_collection_' + randomString(5, undefined, { isLowercase: true }).replace(/\d/g, '');
+    cy.createNamespace(namespace);
+    cy.uploadCollection(collection, namespace);
+    cy.approveCollection(collection, namespace, '1.0.0');
+    cy.navigateTo('hub', Collections.url);
+    cy.get('[data-cy="table-view"]').click();
+    cy.selectTableRow(collection);
+    cy.clickToolbarKebabAction('delete-selected-collections');
+    cy.get('#confirm').click();
+    cy.clickButton(/^Delete collections/);
+    cy.contains(/^Success$/);
+    cy.clickButton(/^Close$/);
+    cy.clickButton(/^Clear all filters$/);
+
+    cy.deleteNamespace(namespace);
+  });
+
+  it('user can delete selected entire collections from repository using the list toolbar', () => {
+    const namespace = 'hub_e2e_namespace_' + randomString(3, undefined, { isLowercase: true });
+    const collection =
+      'hub_e2e_collection_' + randomString(5, undefined, { isLowercase: true }).replace(/\d/g, '');
+    cy.createNamespace(namespace);
+    cy.uploadCollection(collection, namespace);
+    cy.approveCollection(collection, namespace, '1.0.0');
+    cy.navigateTo('hub', Collections.url);
+    cy.get('[data-cy="table-view"]').click();
+    cy.selectTableRow(collection);
+    cy.clickToolbarKebabAction('delete-selected-entire-collections-from-repository');
+    cy.get('#confirm').click();
+    cy.clickButton(/^Delete collections/);
+    cy.contains(/^Success$/);
+    cy.clickButton(/^Close$/);
+    cy.clickButton(/^Clear all filters$/);
 
-  it.skip('user can delete selected entire collections from repository using the list toolbar', () => {});
+    cy.deleteNamespace(namespace);
+  });
 
   it.skip('user can deprecate selected collections using the list toolbar', () => {});
 });
 
 describe('Collections List- Line Item Kebab Menu', () => {
-  let thisCollectionName: string;
+  let collection: string;
   let namespace: string;
   let repository: string;
 
   beforeEach(() => {
-    thisCollectionName = 'hub_e2e_' + randomString(5).toLowerCase();
-    namespace = 'hub_e2e_col_namespace' + randomString(5).toLowerCase();
+    collection = 'hub_e2e_collection_' + randomString(5).toLowerCase();
+    namespace = 'hub_e2e_namespace_' + randomString(5).toLowerCase();
     cy.hubLogin();
     cy.createNamespace(namespace);
-    cy.uploadCollection(thisCollectionName, namespace);
+    cy.uploadCollection(collection, namespace);
     cy.navigateTo('hub', Collections.url);
   });
 
   afterEach(() => {
     if (Cypress.currentTest.title === 'user can deprecate a collection') {
-      cy.undeprecateCollection(thisCollectionName, namespace, repository);
+      cy.undeprecateCollection(collection, namespace, repository);
       cy.galaxykit('task wait all');
     }
-    cy.deleteCollection(thisCollectionName, namespace, repository);
+    if (
+      Cypress.currentTest.title !== 'user can delete entire collection from repository' &&
```

Comment:
Perhaps we should break down this one in different `describe` ones to avoid this kind of logic.